axes/attempts.py

```diff
@@ -130,6 +130,13 @@ def get_user_attempts(request):
     return attempts
 
 
+def reset_user_attempts(request):
+    attempts = _query_user_attempts(request)
+    count, _ = attempts.delete()
+
```

Comment:
also reset cached attempts. same issue as https://github.com/jazzband/django-axes/issues/363

Reply:
I will add this today or tomorrow.

Reply:
disregard, I guess we are already doing this.

Reply:
I noticed, let me know if there is anything else to improve.